libraries/AP_HAL_ChibiOS/hwdef/AET-H743-Basic/hwdef-bl.dat

```diff
@@ -0,0 +1,49 @@
+# hw definition file for processing by chibios_pins.py
+# for AET H743-Basic bootloader
+
+# MCU class and specific type
+MCU STM32H7xx STM32H743xx
+
+# board ID for firmware load
+APJ_BOARD_ID AP_HW_AET-H743-Basic
+
+# crystal frequency, setup to use external oscillator
+OSCILLATOR_HZ 8000000
+
+FLASH_SIZE_KB 2048
+
+# bootloader starts at zero offset
+FLASH_RESERVE_START_KB 0
+
+# the location where the bootloader will put the firmware
+# the H743 has 128k sectors
+FLASH_BOOTLOADER_LOAD_KB 128
+
+
+# order of UARTs (and USB). Allow bootloading on USB and telem1
+SERIAL_ORDER OTG1 UART7
+
+# UART7 (telem1)
+PE7 UART7_RX UART7 NODMA
+PE8 UART7_TX UART7 NODMA
+
+# PA10 IO-debug-console
+PA11 OTG_FS_DM OTG1
+PA12 OTG_FS_DP OTG1
+
+PA13 JTMS-SWDIO SWD
+PA14 JTCK-SWCLK SWD
+
+# make sure Vsw is on during bootloader
+PD10  PINIO1 OUTPUT LOW
+
+PE3 LED_BOOTLOADER OUTPUT LOW
+define HAL_LED_ON 0
+
```

Comment:
I would add the SDcard interface so that the bootloader can update firmware from the SD card

Reply:
I found a wired situation that if we plug SD card for the first time after flashing the board, the board will not boot. So we prefer not adding this feature now.